indico/modules/events/abstracts/controllers/management.py

```diff
@@ -176,6 +176,15 @@ def _process(self):
         return jsonify_form(form)
 
 
+class RHExportBOA(RHManageAbstractsBase):
+    """Export book of abstracts"""
+
+    def _process(self):
+        pdf = AbstractBook(self._target)
```

Comment:
We don't use `self._target` in new code. The legacy Conference object is in `self._conf` (avoid using this as much as possible), the new `Event` object is in `self.event_new`